src/main/java/com/homeaway/devtools/jenkins/testing/WholeClasspathPipelineExtensionDetector.java

```diff
@@ -94,7 +98,11 @@
 
 		HashMap<String, Throwable> failures = new HashMap<>();
 
-		List<String> annotated_classnames = new FastClasspathScanner(_package.orElse("") ).scan().getNamesOfClassesWithAnnotation( _annotation );
+		List<String> annotated_classnames = new ClassGraph()
+				.enableAnnotationInfo()
+				.enableClassInfo()
+				.acceptPackages(_package.orElse(""))
+				.scan().getClassesWithAnnotation( _annotation.getName() ).getNames();
```

Comment:
Extra layer of indentation added here.

```suggestion
		List<String> annotated_classnames = new ClassGraph()
			.enableAnnotationInfo()
			.enableClassInfo()
			.acceptPackages(_package.orElse(""))
			.scan().getClassesWithAnnotation( _annotation.getName() ).getNames();
```